.github/workflows/release.yml

```
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
+      - uses: olafurpg/setup-gpg@v2
+      - run: git fetch --tags --unshallow
+      - name: Publish ${{ github.ref }}
+        run: sbt ci-release docs/docusaurusPublishGhpages
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          GIT_USER: scalameta@scalameta.org
+          GITHUB_DEPLOY_KEY: ${{ secrets.GITHUB_DEPLOY_KEY }}
```

I recall some user saying it GitHub Actions secrets can't start with "GITHUB_". Make sure we are on the latest mdoc version (2.2.4)
```suggestion
          GIT_DEPLOY_KEY: ${{ secrets.GIT_DEPLOY_KEY }}
```

---

I just updated the deploy keys and secrets following instructions here https://scalameta.org/mdoc/docs/docusaurus.html#publish-to-github-pages-from-ci